capstone-concept-template.md

```diff
@@ -0,0 +1,81 @@
+# Capstone Concept Template Document
+
+Copy and paste this document into a gist and then edit it to create your capstone concept document.  This document is an example with the minimum content required for your capstone concept deliverable.  You are free to add further content to help you arrive at your capstone.
+
+# Capstone Concept - AUTHOR NAME
+
+## Problem Statement #1
```

Comment:
I think we could probably just have one problem statement since we'd rather them have _less_ concepts, and three is definitely the max not the requirement.